packages/itwinui-css/src/tile/tile.scss

```diff
@@ -121,6 +138,10 @@
     text-overflow: ellipsis;
   }
 
+  .iui-tile-uploading {
+    font-style: italic;
```

Comment:
amm.. we sure about this?

Reply:
Yes, italics were requested in the original issue.

Reply:
@FlyersPh9 We do not use italics in any other `informational` components. Whats the difference here?

Reply:
We've removed the tile with the blue stripe, we only include positive, warning, negative statuses.  The italics is tied to "upload in progress" which [I would like to separate from disabled state in table as well](https://github.com/iTwin/iTwinUI/pull/697#issuecomment-1184764342).

![Screen Shot 2022-07-14 at 2 26 54 PM](https://user-images.githubusercontent.com/849817/179055734-55f50f39-7b8b-4b32-a29f-18c08a40ced0.png)